### PR TITLE
Add support for Dictionary (and for all other data)

### PR DESCRIPTION
**Pull request by antonGritsenko**
Add examples for Dictionary and JsonExtensionDataAttribute

### CODE REVIEW

**Comment by antonGritsenko** [comment]
Discussion here: https://github.com/mattfrear/Swashbuckle.AspNetCore.Examples/issues/39

File: src/Swashbuckle.AspNetCore.Examples/Examples/ExamplesOperationFilter.cs [hunk 1]
```diff
@@ -37,30 +38,50 @@ private static void SetRequestModelExamples(Operation operation, ISchemaRegistry
...
                     {
                         var definitionToUpdate = schemaRegistry.Definitions[name];
-                        var serializerSettings = SerializerSettings(attr.ContractResolver, attr.JsonConverter);
-
                         definitionToUpdate.Example = FormatJson(provider, serializerSettings, false);
                     }
+                    else
+                    {
+                        // default schema not found, so put example directly to request
+                        request.Schema.Example = FormatJson(provider, serializerSettings, false);
```
Comment (mattfrear) [inline src/Swashbuckle.AspNetCore.Examples/Examples/ExamplesOperationFilter.cs]: Oh, I had forgotten this property existed. I'm not sure if an OperationFilter such as this one should change that property, Perhaps it should be a SchemaFilter instead.

**Comment by mattfrear** [comment]
Thank you for your PR. You can ignore that AppVeyor build failure, I haven't spent any time setting that up correctly.

Can you add unit tests for any new features? I admit the test coverage isn't that great on this project, that is because I created the test infrastructure and wrote the first tests after the code had already been written. But any new code should have tests.

**Comment by mattfrear** [comment]
I have added a unit test to your PR and merged it in. The code has been released https://www.nuget.org/packages/Swashbuckle.AspNetCore.Examples/2.9.0

Thanks for the PR!

**Pull request merged**